crates/bevy_core/src/bytes.rs

```diff
@@ -12,11 +12,7 @@ pub trait Bytes {
 }
 
 /// A trait that indicates that it is safe to cast the type to a byte array reference.
-pub unsafe trait Byteable
-where
-    Self: Sized,
-{
-}
+pub unsafe trait Byteable: Copy + Sized {}
```

Comment:
I definitely prefer this over putting on the individual traits `FromBytes` etc...!
Nice refactor :smile:

Reply:
Agreed!